src/ResourceManagement/Scheduler/Generated/JobsOperationsExtensions.cs

```diff
@@ -0,0 +1,429 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+//
+// Code generated by Microsoft (R) AutoRest Code Generator 1.2.2.0
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+
+namespace Microsoft.Azure.Management.Scheduler.Fluent
+{
+    using Microsoft.Azure;
+    using Microsoft.Azure.Management;
+    using Microsoft.Azure.Management.Scheduler;
+    using Microsoft.Rest;
+    using Microsoft.Rest.Azure;
+    using Microsoft.Rest.Azure.OData;
+    using Models;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    /// <summary>
+    /// Extension methods for JobsOperations.
+    /// </summary>
+    public static partial class JobsOperationsExtensions
+    {
+            /// <summary>
+            /// Gets a job.
+            /// </summary>
+            /// <param name='operations'>
+            /// The operations group for this extension method.
+            /// </param>
+            /// <param name='resourceGroupName'>
+            /// The resource group name.
+            /// </param>
+            /// <param name='jobCollectionName'>
+            /// The job collection name.
+            /// </param>
+            /// <param name='jobName'>
+            /// The job name.
+            /// </param>
+            public static JobDefinitionInner Get(this IJobsOperations operations, string resourceGroupName, string jobCollectionName, string jobName)
+            {
+                return operations.GetAsync(resourceGroupName, jobCollectionName, jobName).GetAwaiter().GetResult();
+            }
```

Comment:
please remove all Sync methods from the generated code. There is a regex expression that should help you.